src/backend/PoetrySlam/schemas.py

```diff
@@ -14,3 +15,23 @@ class User(BaseModel):
 
     class Config:
         orm_mode = True
+
+
```

Comment:
These changes to the schemas are arbitrary and don't serve any purpose right not. also these changes aren't reflected in the database models and isn't propagated through the backend code. I'd suggest removing this.